### ISSUE TITLE
[BITV] 9.1.1.1c/2.1 - Icons of widgets such as "Recommended files" contain decorative images which do not have alternative texts and are not marked as decorative elements with aria-hidden="true". (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#37325]
**This issue was not in a11y report and is related to https://github.com/nextcloud/server/issues/36908 and should be done for 9.1.1.1c/2.1 too.**

![Screenshot from 2023-03-21 16-10-56](https://user-images.githubusercontent.com/6078378/226650679-b17b62f6-1da5-40a5-a0bc-6c88cf8a6dff.png)

- [x] Fix
- [ ] Backport
- [ ] to be added here as todo fort all default-shipped widgets

i.e. ```<div data-v-7d1a0f30="" role="img" class="icon-activity"></div>```

<details>
<summary>Details</summary>

</details>

**Comment by JuliaKirschenheuter** [comment]
Fix:
- [ ] https://github.com/nextcloud/nextcloud-vue/pull/3919

Backport: 
- [ ] Bump new NcVue to stable25
- [ ] Bump new NcVue to stable26

**Comment by JuliaKirschenheuter** [comment]
New NcVue version and bumps are not ready yet. Reopening issue.

**Comment by JuliaKirschenheuter** [comment]
> New NcVue version and bumps are not ready yet. Reopening issue.

not needed anymore
